langchain/prompts/chat.py

```diff
@@ -143,13 +143,19 @@ class ChatPromptTemplate(BaseChatPromptTemplate, ABC):
     input_variables: List[str]
     messages: List[Union[BaseMessagePromptTemplate, BaseMessage]]
 
+    @classmethod
+    def from_template(cls, template: str, **kwargs: Any) -> ChatPromptTemplate:
```

Comment:
This class method makes it sound like it should be taking a prompt template as an input rather than a string. One possibility is to make a flexible interface here that will be able to handle multiple types for convenience.